palette/tests/convert/data_color_mine.rs

```diff
@@ -165,9 +178,19 @@ fn check_equal_cie(src: &ColorMine, tgt: &ColorMine) {
     assert_relative_eq!(src.xyz, tgt.xyz, epsilon = 0.05);
     assert_relative_eq!(src.yxy, tgt.yxy, epsilon = 0.05);
 
+    //these transformations are problematic and need bigger epsilon
+    assert_relative_eq!(src.lab, tgt.lab, epsilon = 6.0);
+
     // hue values are not passing for from_yxy conversion. Check github #48 for
     // more information assert_relative_eq!(src.lch.hue, tgt.lch.hue, epsilon =
     // 0.05);
+    //low chroma colors have hues that are hard to calculate precisely
+    
+    if src.lch.chroma > 20.0 {
+        println!("{}", src.lch.chroma);
+        assert_relative_eq!(src.lch, tgt.lch, epsilon = 10.0);
+    }
```

Comment:
We could set hue to 0 (or the expected value) for low chroma cases to sort of "ignore" it. That way the remaining components will still be checked.